### PR TITLE
added front-end for items

### PR DESCRIPTION
**Pull request by faesalm**

### CODE REVIEW

**Comment by faesalm** [comment]
@Tetheta  super simple frontend stuff

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -29,6 +29,15 @@ function setPageTitle() {
   document.title = parameterUsername + " - User Page";
 }
 
+/** Sets the Item page link to point to user ad*/
+function setItemLink() {
+  var item_link = document.getElementById("item-link");
```
Comment (Tetheta) [inline src/main/webapp/js/user-page-loader.js]: const not var is preferred

File: src/main/webapp/js/item_page_loader.js [hunk 2]
```diff
@@ -15,10 +19,16 @@ function createMap(user_lat, user_lng, user_title) {
 }
 
 function buildUI() {
+  // URL must include ?user=XYZ parameter. If not, redirect to homepage.
```
Comment (Tetheta) [inline src/main/webapp/js/item_page_loader.js:22]: So if you're using the user to show their posting, what happens when they want to have multiple postings? 
Also this gives away information in the URL about the poster (probably fine in this case).

This is fine for now, but keep in mind you'll have to adjust things if you want to scale :) One posting per user is a good starting point though

**Review by Tetheta — DISMISSED** [review]

File: src/main/webapp/js/item_page_loader.js (hunk 2 above)
Comment (faesalm) [reply src/main/webapp/js/item_page_loader.js:22]: Yes for sure. This is already considered in a TO-DO. A lot of the code is based on one posting per user (for example location is fetched from user. It is not in the actual item). Will rework if we have time.

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (faesalm) [reply src/main/webapp/js/user-page-loader.js]: done

**Pull request merged**